### PR TITLE
feat(serverless): add headers to examples

### PR DESCRIPTION
**Pull request by thomas-tacquet**
### Your checklist for this pull request

- [X] Check that the commit messages match our requested structure.
- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
According to the other documentation and examples, this PR refreshed available examples in documentation for best practices.

### CODE REVIEW

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

File: compute/functions/reference-content/code-examples.mdx [hunk 1]
```diff
@@ -244,7 +252,10 @@ return {
 ```js
 module.exports.myHandler = (event, context, callback) => {
   return {
-    message: "message",
+    body: "message",
+    headers: {
+      "Content-Type": "text/plain",
```
Comment (norbjd) [inline compute/functions/reference-content/code-examples.mdx:271]:
```suggestion
      "Content-Type": ["text/plain"],
```
?

File: compute/functions/reference-content/code-examples.mdx [hunk 2]
```diff
@@ -386,84 +400,93 @@ Please refer to our guide on [code packaging](https://www.scaleway.com/en/docs/c
...
         });
 
-        req.on('error', (e) => {
-          reject({
-              statusCode: 500,
-              body: 'Something went wrong!'
+        res.on("end", () => {
+          resolve({
+            statusCode: 200,
+            headers: { "Content-Type": "application/json" },
```
Comment (norbjd) [inline compute/functions/reference-content/code-examples.mdx:433]:
```suggestion
            headers: { "Content-Type": ["application/json"] },
```

File: compute/functions/reference-content/code-examples.mdx [hunk 3]
```diff
@@ -386,84 +400,93 @@ Please refer to our guide on [code packaging](https://www.scaleway.com/en/docs/c
...
-    hostname: 'postman-echo.com',
-    path: '/post',
-    method: 'POST',
+    hostname: "postman-echo.com",
+    path: "/post",
+    method: "POST",
     headers: {
-      'Content-Type': 'application/json'
-    }
+      "Content-Type": "application/json",
```
Comment (norbjd) [inline compute/functions/reference-content/code-examples.mdx:471]: not necessary (here it's request options?)

File: compute/functions/reference-content/code-examples.mdx [hunk 4]
```diff
@@ -386,84 +400,93 @@ Please refer to our guide on [code packaging](https://www.scaleway.com/en/docs/c
...
-          });
+    const req = https.request(options, (res) => {
+      res.on("data", (chunk) => {
+        dataString += chunk;
+      });
+
+      res.on("end", () => {
+        resolve({
+          statusCode: 200,
+          headers: { "Content-Type": "application/json" },
```
Comment (norbjd) [inline compute/functions/reference-content/code-examples.mdx:484]:
```suggestion
          headers: { "Content-Type": ["application/json"] },
```

File: compute/functions/reference-content/code-examples.mdx [hunk 5]
```diff
@@ -481,14 +504,16 @@ According to the official documentation, to use ES modules you can specify the m
 This then enables you to write your code for ES modules:
 
 ```javascript
-export {handle};
+export { handle };
+
+function handle(event, context, cb) {
+  return {
+    body: process.version,
+    headers: { "Content-Type": "text/plain" },
```
Comment (norbjd) [inline compute/functions/reference-content/code-examples.mdx:526]:
```suggestion
    headers: { "Content-Type": ["text/plain"] },
```

**Review by norbjd — CHANGES_REQUESTED** [review]

**Review by norbjd — APPROVED** [review]

**Pull request merged**